frameworks/cassandra/universe/config.json

```diff
@@ -38,7 +38,7 @@
         "detect_zones": {
           "description": "Whether to use Zones when available for Cassandra racks.  This value cannot be changed after initial deployment",
           "type": "boolean",
-          "default": true
+          "default": false
```

Comment:
@mpereira should a similar change be made for elastic?

Reply:
It was done in the original PR https://github.com/mesosphere/dcos-commons/pull/1855/commits/4f7e5780295c059794c1798b7ec8bd6225ee229f